lib/core/network/api/services/remote_config_service.dart

```diff
@@ -2,9 +2,25 @@ import 'dart:convert';
 
 import 'package:firebase_remote_config/firebase_remote_config.dart';
 
-enum Networks { telos, telosTestnet, eos, eosTestnet }
+enum Network {
+  telos('telos'),
+  telosTestnet('telosTestnet'),
+  eos('eos'),
+  eosTestnet('eosTestnet');
 
```

Comment:
Why do you need to do that?

If you just left it as it was :
enum Networks { telos, telosTestnet, eos, eosTestnet }

Then you can do Networks.telos.name and that will be == "telos"

https://api.flutter.dev/flutter/dart-core/EnumName/name.html

I'd change this back to what it was.

Reply:
BTW I have used the .name function in the code so I know it works.

Reply:
Edit: Also yes I am in favor or renaming it to "Network" without the 's' - that's what it should be. 

Thanks for that ;)